frontend/src/layout/Footer.tsx

```diff
@@ -10,7 +10,7 @@ export const Footer: React.FC = () => {
     const { t, i18n } = useTranslation();
 
     return (
-        <footer css={{
+        <footer role="contentinfo" css={{
```

Comment:
Honestly, I would just not change this for now. Especially after reading your commit message and the part of the report again, I'm really not sure why this would be necessary. Also for Chrome, "footer" sounds like a way better description than "content info". And I also don't see a reason why JAWS wouldn't be able to understand the semantic element `<footer>`. 

So what I'd suggest is: drop this commit and lets talk to the report people again to clear this up.